Assets/Scripts/Player/Player.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour {
+
+	public int currentHealth;
+	public int startingHealth = 30;
+
+	public int currentMana;
+	public int manaThisTurn; // Increases by one every turn.
+	public int startingMana = 3;
+	public int maxMana = 7;
+
+	public Deck deck;
+	public List<Card> hand;
+
+	public int startingHandSize = 4;
+	public int maxHandSize;
+
+	// Events!
+	// @TODO: Figure out how to implement these as GameActions?
+	public object whenTurnIsStarted;
```

Comment:
`whenTurnIsStarted` should probably be an event on each card - then when the turn starts for a player we can just loop over all their cards and call that event